frontend/packages/console-shared/src/components/spotlight/Spotlight.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes expand {
+  0% {
+    outline-offset: -4px;
+    outline-width: 4px;
+    opacity: 1;
+  }
+  100% {
+    outline-offset: 21px;
+    outline-width: 12px;
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fade-out {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+.ocs-spotlight {
+  pointer-events: none;
+  position: absolute;
+  &__with-backdrop {
+    mix-blend-mode: hard-light;
```

Comment:
This causes the blue for the guided tour to be slightly a different color than the one we actually want. Perhaps raise a separate issue for this since it was an issue before as well.
![image](https://user-images.githubusercontent.com/14068621/100131519-218cbe80-2e52-11eb-9ae0-3afb4ce7a2ff.png)

Reply:
raised issue https://issues.redhat.com/browse/ODC-5178